kpatch/kpatch

```diff
@@ -165,7 +165,27 @@ load_module () {
 	fi
 
 	echo "loading patch module: $module"
-	insmod "$module"
+	local i=0
+	while true; do
+		out=$(insmod "$module" 2>&1)
+		[[ -z $out ]] && break
+		echo $out 1>&2
+		[[ ! $out =~ "Device or resource busy" ]] &&
```

Comment:
I have no i18n experience, but is it assured that insmod output will be in English?  (Too bad it doesn't propagate out the error code as insmod exit status.)